src/analysis/special_functions/integrals.lean

```diff
@@ -409,7 +409,7 @@ lemma integral_sin_pow_aux :
     + (n + 1) * (∫ x in a..b, sin x ^ n) - (n + 1) * ∫ x in a..b, sin x ^ (n + 2) :=
 begin
   let C := sin a ^ (n + 1) * cos a - sin b ^ (n + 1) * cos b,
-  have h : ∀ α β γ : ℝ, α * (β * α * γ) = β * (α * α * γ) := λ α β γ, by ring,
+  have h : ∀ α β γ : ℝ, (β * α * γ) * α = β * (α * α * γ) := λ α β γ, by ring,
```

Comment:
What's the motivation for this change?

Reply:
This is because in order for integration by parts (`integral_mul_deriv_eq_deriv_mul`) to be valid for functions valued in a possibly non-commutative Banach algebra, one has to state it as
`∫ x in a..b, u x * v' x = u b * v b - u a * v a - ∫ x in a..b, u' x * v x`,
rather than
`∫ x in a..b, u x * v' x = u b * v b - u a * v a - ∫ x in a..b, v x * u' x`
which was the statement currently in the library for real-valued functions. Changing this needs a slight adjustment to the two places in `special_functions/integrals.lean` where `integral_mul_deriv_eq_deriv_mul` is used.

Reply:
I completely missed that you swapped the multiplication order in that lemma, sorry for the confusion!